fmcg_store/data/knowledge_article.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="knowledge_article_5" model="knowledge.article">
+        <field name="name">Grocery Shop</field>
+        <field name="internal_permission">write</field>
+        <field name="cover_image_id" ref="knowledge_cover_4"/>
+        <field name="is_article_visible_by_everyone" eval="True"/>
+        <field name="is_locked" eval="True"/>
+        <field name="body">
+            <![CDATA[
+                     <p>
+                        <strong><span style="font-size: 36px;">The Grocery Shop</span></strong>
+                    </p>
+                    <p>
+                        Setting up a Point of Sale (POS) system for a grocery store which is essential for efficiently managing various product categories, handling transactions, and providing excellent customer service. Grocery stores are businesses that carry a large selection of products like Food Grains, Oil &amp; Clarified Butter, Dry Fruits, Chocolates, Snacks &amp; Packaged Food with multiple variants etc…Here's a step-by-step guide to setting up a POS system for a grocery store, also known as a Grocery Store, Supermarket, or Food Store:
+                    </p>
+                    <p>In the retail store, we set up the POS module for B2C customers and sales for B2B customers. They use the Sales, Purchase, Inventory, Accounting, Contact, and Employee modules.</p>
+                    <div data-oe-protected="true" class="o_knowledge_behavior_anchor o_knowledge_behavior_type_toc">
+                        <div class="o_knowledge_toc_content">
+                            <a href="#" data-oe-nodeid="0" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Flow 1 : Purchase Flow.</a>
+                            <a href="#" data-oe-nodeid="1" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Flow 2: Point of sale - Regular flow with Buy one get one free product</a>
+                            <a href="#" data-oe-nodeid="2" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Flow 3: Point of sale - Customer Account </a>
+                            <a href="#" data-oe-nodeid="3" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Flow 4: Point of Sale - Ship Later &amp; Global discount @10% </a>
+                            <a href="#" data-oe-nodeid="4" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Flow 5: Sale Flow – B2B Customer</a>
+                        </div>
+                    </div>
+                    <br/>
```

Comment:
```suggestion
```
H2 tag does keep enough margin so no need for a BR tag 
Please remove it from the whole knowledge article where it does not make any sense